Tests/IntegrationTests/Actions Tests/Redirect/RedirectComponentTests.swift

```diff
@@ -292,7 +292,12 @@ class RedirectComponentTests: XCTestCase {
         }
         delegate.onDidFail = { _, _ in XCTFail("Should not call onDidFail") }
         
-        let action = RedirectAction(url: URL(string: "https://google.com")!, paymentData: nil, nativeRedirectData: "test_nativeRedirectData")
+        let action = RedirectAction(
+            url: URL(string: "https://google.com")!,
+            paymentData: nil,
+            type: .nativeRedirect,
+            nativeRedirectData: "test_nativeRedirectData"
```

Comment:
[Documentation](https://docs.adyen.com/api-explorer/Checkout/71/post/payments#responses-200-action-CheckoutNativeRedirectAction-nativeRedirectData) says it's just a string with no further explanation. Is this an URL or anything else? Should we have tests for this string?